src/docs-ui/app/app.component.html

```diff
@@ -3,6 +3,38 @@
     <div td-toolbar-content layout="row" layout-align="start center" flex>
       <span flex></span>
 
+      <vui-account-profile name="donald duck" email="donald.duck@teradata.com">
+        <ng-container account-info-list>
+          <mat-list-item (click)="_blockEvent($event)">
+            <mat-icon matListAvatar>account_balance</mat-icon>
+            <span matLine class="mat-body-1">Default</span>
+            <span matLine>organization</span>
+          </mat-list-item>
+        </ng-container>
+        <ng-container account-action-list>
+          <button
```

Comment:
This theme toggling is going to be something that will be common. Can we make this re-useable? Either as in a re-usable component or maybe an input `themeToggle: boolean`?

Reply:
yes makes sense to do that as a directive for the button. will do that as a separate effort not hold this PR up